moveit_ros/moveit_servo/launch/servo_example.launch.py

```diff
@@ -39,6 +39,17 @@ def generate_launch_description():
     servo_yaml = load_yaml("moveit_servo", "config/panda_simulated_config.yaml")
     servo_params = {"moveit_servo": servo_yaml}
 
+    # Get the ee_frame_name parameter
```

Comment:
Nice use of launch parameters. With this being said it is necessary here as `servo.yaml` contains the parameter `ee_frame_name` already.